lib/praxis/response_definition.rb

```diff
@@ -238,17 +238,36 @@ def validate_headers!(response)
     def validate_content_type!(response)
       return unless media_type
 
-      extracted_identifier = nil
+      response_content_type = {}
       if response.headers['Content-Type']
-        extracted_identifier = Praxis::ContentTypeParser.parse(response.headers['Content-Type'])[:type]
+        response_content_type = Praxis::ContentTypeParser.parse(response.headers['Content-Type'])
       end
 
-      if media_type.identifier != extracted_identifier
```

Comment:
We could move that all to a separate `MediaTypeIdentifier` type and optimize it there. See #135 for more.